jupyter_server/event_schemas/gateway_client/v1.yaml

```diff
@@ -0,0 +1,40 @@
+"$id": https://events.jupyter.org/jupyter_server/gateway_client/v1
+version: 1
+title: Gateway Client activities.
+personal-data: true
+description: |
+  Record events of a gateway client.
+type: object
+required:
+ - status
+ - message
+properties:
+  status:
+    enum:
+      - error
+      - success
+    description: |
+      Status received by Gateway client based on the rest api operation to gateway kernel.
+
+      This is a required field.
+
+      Possible values:
+
+      1. error
+         Error response from a rest api operation to gateway kernel.
+
+      2. success
+         Success response from a rest api operation to gateway kernel.
+  status_code:
+    type: number
+    description: |
+      Http response codes from a rest api operation to gateway kernel.
+      Examples: 200, 400, 502, 503, 599 etc.
+  message:
```

Comment:
"message" is a (undocumented, unfortunately) disallowed key in jupyter_events. We probably need to change this to something like "msg".

Reply:
Done, changed it to `msg` to be in sync with `kernel_actions`